AppSettingsManagerBff/AppSettingsManagerBff.Infrastructure/ApiRepositories/HttpUserRepository.cs

```diff
@@ -43,4 +43,45 @@ public async Task<ApiBaseUser> CreateUser(string userId, string password)
         // this is just a way to suppress errors from there being no return value on this method
         throw new NotImplementedException();
     }
-}
+
+    public async Task<ApiBaseUser> UpdateUser([FromRoute][Required] UpdateUserRequest request)
+    {
+        //Serialize request object to string
+        var serializedRequest = JsonSerializer.Serialize(request);
+
+        //Wrap request object in StringContent object
+        var content = new StringContent(serializedRequest, System.Text.Encoding.UTF8, "application/json");
+
+        //Send Request
+        var response = await _httpClient.PutAsync($"{request.userId}/{request.password}", content);
```

Comment:
Check out the routing and parameters on Nahiyan's controller method https://github.com/WSU-4110/AppSettingsManager/blob/8afe2f771ec2e97047b3c270cd5331bea1f5dc21/AppSettingsManagerAPI/AppSettingsManagerApi/Controllers/UserController.cs#L33

The way it's set up right now, there's actually no request object, you're just sending a username and password
```suggestion
        var response = await _httpClient.PutAsync($"userId/{request.userId}/password/{request.password}");
```